src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,35 @@
+.calendar {
+  margin: 0 auto;
+  width: $cell-width * 7 + (6 * $gap);
```

Comment:
What do `7` and `6` mean? Don't use mystery numbers. Create variables for it. And move calculations to a variable to not clutter the styles code.